ssc/cmod_utilityrate5.cpp

```diff
@@ -2203,15 +2209,18 @@ class cm_utilityrate5 : public compute_module
 					dollars_applied += monthly_cumulative_excess_dollars[m - 1];
 				}
 
-				if (monthly_ec_charges[m] < 0)
-				{
-					payment[c - 1] -= monthly_cumulative_excess_dollars[m] + monthly_ec_charges[m];
-					dollars_applied += monthly_cumulative_excess_dollars[m] + monthly_ec_charges[m];
-					monthly_cumulative_excess_dollars[m] = -monthly_ec_charges[m];
+                if (monthly_ec_charges[m] < 0)
```

Comment:
We need the ssc test and SAM to show the hourly and subhourly payments matching with the monthly roll up values and ultimately the annual energy savings value.